src/main/java/ei/algobaroapi/domain/room/service/RoomServiceImpl.java

```diff
@@ -70,8 +79,46 @@ public RoomDetailResponseDto getRoomByRoomUuid(String roomShortUuid) {
     }
 
     @Override
-    public void startCodingTest(Long roomId) {
-        // TODO: RoomStatus Running 으로 변환
+    @Transactional
+    public RoomDetailResponseDto startCodingTest(String roomShortUuid) {
+        // Room 찾기
+        Room room = roomRepository.findByRoomUuidStartingWith(roomShortUuid)
+                .orElseThrow(() -> RoomNotFoundException.of(RoomErrorCode.ROOM_NOT_FOUND));
+
+        Long roomId = room.getId();
+
+//        // RoomMember 찾기
+        List<RoomMember> findRoomMembers = roomMemberRepository.findByRoomId(roomId);
+
+        // RoomMember 들이 준비 상태 인지 확인
+        for (RoomMember roomMember : findRoomMembers) {
+            if (!checkRoomMemberIsReady(roomMember)) {
+                throw RoomNotFoundException.of(RoomErrorCode.ROOM_NOT_READY);
+            }
+        }
```

Comment:
```suggestion
        List<RoomMember> findRoomMembers = roomMemberService.getByRoomIdAllReady(roomId);
```

식으로 RoomMemberService에서 조회 및 검증을 수행시킬 수 있을 것 같아요

Reply:
확실히 RoomMember 조회 및 준비 상태를 검증하는 것은 RoomMember 도메인쪽에서 진행하는 것이 좋겠네요! 감사합니다 ㅎㅎㅎ